docs/child-applications.md

```diff
@@ -143,6 +140,41 @@ export const unmount = [
 ];
 ```
 
+### unload
+The `unload` lifecycle is an optionally implemented lifecycle function. It will be called whenever an application should be
```

Comment:
I recommend reading the docs first because they explain why the `unload` feature exists.